website/docs/reference/appsmith-framework/widget-actions/navigate-to.md

```diff
@@ -29,7 +29,7 @@ Page name or URL to which you would like to be transported. `PageName` is case-s
 
 <dd>
 
-Query parameters passed via the URL. Used to share information with the destination page. See [Sharing data via query params](/advanced-concepts/sharing-data-across-pages#sharing-data-via-query-params).
+Query parameters passed via the URL. Used to share information with the destination page. It is important to use triple braces ({{{ }}}) to ensure that the parameters are encoded correctly as a part of the URL. Even if the value is a static string, the triple braces are required to include the parameter in the navigation. See [Sharing data via query params](/advanced-concepts/sharing-data-across-pages#sharing-data-via-query-params).
```

Comment:
I don't think "triple braces" is a known concept and this is also not something we would want to to introduce to our users. Syntactically, this is just a combination of two different concepts:
1. A JS binding: `{{ }}`
2. An object: `{"key": "value"}`